auslib/db.py

```diff
@@ -1596,13 +1598,22 @@ def getRulesMatchingQuery(self, updateQuery, fallbackChannel, transaction=None):
            For cases where a particular updateQuery channel has no
            fallback, fallbackChannel should match the channel from the query."""
 
+        # TODO: is there a better place to put this?
+        versionClass = MozillaVersion
+        if updateQuery["product"] == "Guardian":
+            versionClass = LooseVersion
```

Comment:
Hmm, a helper function seems like a good idea. It makes things a bit more readable, at least.

If we put something in the config, we still need to pass the config down to the database layer, so I don't think it will help readability. Though, arguably it's a more obvious place for it to live, rather than buried down in a helper module.